Alignment/HIPAlignmentAlgorithm/python/common_cff_py.txt

```diff
@@ -1,89 +1,227 @@
-from CondCore.DBCommon.CondDBSetup_cfi import *
-
-# loading magnetic field and geometry
-process.load("Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff")
-process.load("Configuration.Geometry.GeometryRecoDB_cff")
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("Alignment")
+process.load("Alignment.CommonAlignmentProducer.AlignmentTrackSelector_cfi")
+process.load("RecoTracker.FinalTrackSelectors.TrackerTrackHitFilter_cff")
+#process.load("RecoTracker.TransientTrackingRecHit.TransientTrackingRecHitBuilderWithoutRefit_cfi")
+process.load("RecoTracker.TrackProducer.TrackRefitters_cff")
+process.load("RecoVertex.BeamSpotProducer.BeamSpot_cff")
+process.load('Configuration.Geometry.GeometryRecoDB_cff')
+
+#Tracker
+from RecoTracker.GeometryESProducer.TrackerRecoGeometryESProducer_cfi import *
+
+#Muon
+from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *
+from RecoMuon.DetLayers.muonDetLayerGeometry_cfi import *
+
+#  Alignment
+from Geometry.TrackerGeometryBuilder.idealForDigiTrackerGeometry_cff import *
+from Geometry.CSCGeometryBuilder.idealForDigiCscGeometry_cff import *
+from Geometry.DTGeometryBuilder.idealForDigiDtGeometry_cff import *
+process.load("Configuration.StandardSequences.MagneticField_cff")
+#process.load("Configuration.StandardSequences.MagneticField_38T_cff")
+#process.load("Configuration.StandardSequences.MagneticField_0T_cff")
 
```

Comment:
@CandiceYou please remove these commented lines to avoid the impression that one needs to choose the correct line depending on the magnetic field which one is using.